### PR TITLE
[FIX][12.0][account_move_transactionid_import] s/transaction_ref/ref

### PR DESCRIPTION
**Pull request by Tonow-c2c**
the field `transaction_ref` doesn't exist any more

### CODE REVIEW

**Review by grindtildeath — APPROVED** [review]

**Review by grindtildeath — CHANGES_REQUESTED** [review]
Doesn't work as expected, let's check it out together when you're back.

**Comment by grindtildeath** [comment]
@Tonow-c2c https://github.com/Tonow-c2c/account-reconcile/pull/1 :wink:

File: account_move_base_import/parser/file_parser.py [hunk 1]
```diff
@@ -70,13 +72,24 @@ def _parse(self, *args, **kwargs):
...
+        if self.parsed_file is None:
+            if self.ftype == 'csv':
+                self.parsed_file = self._parse_csv()
+            else:
+                self.parsed_file = self._parse_xls()
+        if self.support_multi_moves:
+            if len(self.parsed_file) <= self.current_line:
+                return False
+            else:
+                print(self.current_line)
```
Comment (i-vyshnevska) [inline account_move_base_import/parser/file_parser.py]:
```suggestion
```

File: account_move_transactionid_import/parser/transactionid_file_parser.py [hunk 2]
```diff
@@ -32,6 +32,7 @@ def __init__(self, profile, ftype='csv', extra_fields=None, header=None,
         super().__init__(
             profile, extra_fields=conversion_dict, ftype=ftype, header=header,
             **kwargs)
+        self.support_multi_moves = True
```
Comment (i-vyshnevska) [inline account_move_transactionid_import/parser/transactionid_file_parser.py:45]: should be optional?

Comment (i-vyshnevska) [reply account_move_transactionid_import/parser/transactionid_file_parser.py:45]: I don't really understand the message of one commit:
 "parser allows to generate 1 account.move for each line in the imported file"
account move should have at least 2 lines, and second is searched/created on statement reconciliation 
Lint need to be checked

Comment (grindtildeath) [reply account_move_transactionid_import/parser/transactionid_file_parser.py:45]: > should be optional?

Not when using transaction_id parser due to the change from #292 

> "(support_multi_moves) parser allows to generate 1 account.move for each line in the imported file"
account move should have at least 2 lines, and second is searched/created on statement reconciliation

The `support_multi_moves` flag on the parser allows to generate 1 account.move per line in the imported statement. However resulting account move (for one line) will always include at least two move lines (cf https://github.com/OCA/account-reconcile/blob/12.0/account_move_base_import/models/account_journal.py#L319-L324) and even 3 if there's a commission amount (cf https://github.com/OCA/account-reconcile/blob/12.0/account_move_base_import/models/account_journal.py#L325-L326).

@Tonow-c2c Could you fix the lint and commit @i-vyshnevska 's suggestion ?

File: account_move_base_import/models/account_journal.py [hunk 3]
```diff
@@ -245,7 +245,6 @@ def prepare_move_line_vals(self, parser_vals, move):
             'debit_cash_basic': values['debit'],
             'credit_cash_basic': values['credit'],
             'balance_cash_basic': values['debit'] - values['credit'],
```
Comment (grindtildeath) [inline account_move_base_import/models/account_journal.py]: @Tonow-c2c Could you please also remove these undefined fields ?

**Review by vrenaville — APPROVED** [review]

**Review by i-vyshnevska — APPROVED** [review]

**Review by grindtildeath — APPROVED** [review]

**Comment by simahawk** [comment]
@yvaucher quick check and merge?

**Pull request merged**